### DISCUSSION TITLE
How hard is it to add CD-Audio support for CD-ROM emulation devices

### DISCUSSION THREAD

**Discussion by AiGeekNet** [akuker/RASCSI#1007] [Ideas]
I noticed a feature comparison of several popular SCSI emulation projects, The MacHD project especially supports CD Audio simulation, RaSCSI has the most comprehensive functions but only lacks this function, CD Audio emulation must support CUE/BIN disc image format, is it unique or not suitable for Raspberry Pi?

Lack of CD Audio support, many CD-ROM games can't be used, and the vintage computer lacks a multimedia capability that would make ordinary people think it is cool.

**Comment by rdmark** [comment]
cue/bin support is discussed in https://github.com/PiSCSI/piscsi/issues/535

This is only the first step towards redbook audio though. It is not a trivial project and would require a lot of research, coding and testing to get it to work right. If someone would like to contribute their efforts and code we would happily consider it, of course!
